r6p2/src/devicedrv/mali/linux/mali_osk_time.c

```diff
@@ -54,6 +54,6 @@ u64 _mali_osk_time_get_ns(void)
 u64 _mali_osk_boot_time_get_ns(void)
 {
 	struct timespec tsval;
-	get_monotonic_boottime(&tsval);
+	getboottime(&tsval);
```

Comment:
What is it fixing exactly? On which kernel, with which version?

Reply:
for 4.19-rc2 kernel, its for our build (Orange Pi Win Plus A64 )

Reply:
get_monotonic_boottime has been disabled, in this patch https://lore.kernel.org/patchwork/patch/978060/